doc/radosgw/lua-scripting.rst

```diff
@@ -306,11 +314,29 @@ Operations Log
 ~~~~~~~~~~~~~~
 The ``Request.Log()`` function prints the requests into the operations log. This function has no parameters. It returns 0 for success and an error code if it fails.
 
+Tracing
+~~~~~~~
+Tracing functions can be used only in `postRequest` context.
+
+- ``Request.Trace.SetAttribute()`` - sets the attribute for the request's trace.
+  Takes two arguments. The first is the `key`, which should be a string. The second is the value, which can either be a string or a number.
```

Comment:
First bit of line 322 isn’t a sentence.  Suggest

“the request’s trace, and takes two arguments.”

Also if \`key\` is in backticks, shouldn’t \`value\` be as well?  

Maybe clarify if “number” means an integer, a real, or either?